services/src/main/java/org/keycloak/forms/login/freemarker/model/WebAuthnAuthenticatorsBean.java

```diff
@@ -34,22 +42,40 @@ public WebAuthnAuthenticatorsBean(KeycloakSession session, RealmModel realm, Use
                 .map(WebAuthnCredentialModel::createFromCredentialModel)
                 .map(webAuthnCredential -> {
                     String credentialId = Base64Url.encodeBase64ToBase64Url(webAuthnCredential.getWebAuthnCredentialData().getCredentialId());
-                    String label = (webAuthnCredential.getUserLabel()==null || webAuthnCredential.getUserLabel().isEmpty()) ? "label missing" : webAuthnCredential.getUserLabel();
-                    return new WebAuthnAuthenticatorBean(credentialId, label);
+                    String label = (webAuthnCredential.getUserLabel() == null || webAuthnCredential.getUserLabel().isEmpty()) ? "label missing" : webAuthnCredential.getUserLabel();
+                    String createdAt = getDateTimeFromMillis(realm, webAuthnCredential.getCreatedDate());
+                    return new WebAuthnAuthenticatorBean(credentialId, label, createdAt);
                 }).collect(Collectors.toList());
     }
 
+    private static String getDateTimeFromMillis(RealmModel realm, long millis) {
```

Comment:
This method might be used as a utility method that can be used not only webauthn UI but also other parts.

However, as you've mentioned, the current keycloak's UI does not use such the format. But it can be used in the future.

Reply:
Agree.

Reply:
Thanks.